data/seeds/04-client_notes.js

```diff
@@ -0,0 +1,340 @@
+          created_by: '6',
+        },
+        {
+          client_id: 3,
+          source_view: 'caseview',
+          message:
+            'Nisi parturient tortor Congue Dictum. Adipiscing sit in risus. Etiam.',
+          created_by: '6',
+        },
+        {
+          client_id: 3,
+          source_view: 'caseview',
+          message:
+            'Proin sollicitudin varius ultrices hac sem Pharetra nibh leo dapibus ligula placerat egestas consequat mi. Curae; curae; suscipit convallis tellus dictum aliquam diam orci ad. Nam risus eu quis massa, a praesent gravida nam iaculis hendrerit malesuada bibendum id phasellus.',
+          created_by: '6',
+        },
+        {
+          client_id: 3,
+          source_view: 'caseview',
+          message:
+            'Dolor lacus feugiat morbi tortor vulputate pretium nisl in luctus integer sem mi sociis vulputate vehicula varius. Platea nam volutpat.',
+          created_by: '6',
+        },
+        {
+          client_id: 3,
+          source_view: 'caseview',
+          message:
+            'Vehicula hac curae;. Pretium congue, et natoque ante lorem vivamus turpis magna commodo nisi. Sed sem mauris habitant sagittis nulla justo felis magna adipiscing pellentesque pede litora cum habitasse rhoncus.',
+          created_by: '6',
+        },
+        {
+          client_id: 4,
+          source_view: 'caseview',
+          message:
+            'Vehicula hac curae;. Pretium congue, et natoque ante lorem vivamus turpis magna commodo nisi. Sed sem mauris habitant sagittis nulla justo felis magna adipiscing pellentesque pede litora cum habitasse rhoncus.',
+          created_by: '8',
+        },
+        {
+          client_id: 4,
+          source_view: 'caseview',
+          message:
+            'Nibh hac. Taciti blandit dapibus ullamcorper. Maecenas sit netus diam.',
+          created_by: '8',
+        },
+        {
+          client_id: 4,
+          source_view: 'caseview',
+          message:
+            'Proin sollicitudin varius ultrices hac sem Pharetra nibh leo dapibus ligula placerat egestas consequat mi. Curae; curae; suscipit convallis tellus dictum aliquam diam orci ad. Nam risus eu quis massa, a praesent gravida nam iaculis hendrerit malesuada bibendum id phasellus.',
+          created_by: '8',
+        },
+        {
+          client_id: 5,
+          source_view: 'caseview',
+          message:
+            'Proin sollicitudin varius ultrices hac sem Pharetra nibh leo dapibus ligula placerat egestas consequat mi. Curae; curae; suscipit convallis tellus dictum aliquam diam orci ad. Nam risus eu quis massa, a praesent gravida nam iaculis hendrerit malesuada bibendum id phasellus.',
+          created_by: '8',
+        },
+      ]);
+    });
+};
```

Comment:
Everything looks good!

Reply:
Both seed files are looking great and ready to use!